discovery-infra/tests/test_validations.py

```diff
@@ -7,7 +6,16 @@
 from assisted_service_client.rest import ApiException
 
 
+@pytest.mark.validations
 class TestValidations(BaseTest):
+    def _wait_and_kill_installer(self, cluster, nodes, host):
+        # Wait for specific host to be in installing in progress
+        cluster.wait_for_specific_host_status(host=host,
+                                              statuses=[consts.NodesStatus.INSTALLING_IN_PROGRESS])
+        # Kill master installer to simulate host error
+        selected_node = nodes.get_node_from_cluster_host(host)
+        selected_node.kill_podman_container_by_name("assisted-installer")
```

Comment:
maybe came time to have node.kill_installer?

Reply:
Fixed